src/groundlight/client.py

```diff
@@ -200,16 +254,15 @@ def submit_image_query(  # noqa: PLR0913 # pylint: disable=too-many-arguments
         """
         if wait is None:
             wait = self.DEFAULT_WAIT
+        if patience_time is None:
+            patience_time = self.DEFAULT_PATIENCE
 
         detector_id = detector.id if isinstance(detector, Detector) else detector
 
         image_bytesio: ByteStreamWrapper = parse_supported_image_types(image)
 
         params = {"detector_id": detector_id, "body": image_bytesio}
-        if wait == 0:
-            params["patience_time"] = self.DEFAULT_WAIT
-        else:
-            params["patience_time"] = wait
+        params["patience_time"] = patience_time
```

Comment:
I think we'd be better off leaving this blank if it wasn't set in python.  This will have the effect of overriding the server-configured patience time to 30, even if the customer has customized it on the website, if they don't include patience when calling the SDK.